### DISCUSSION TITLE
support use single quotes instead of double quotes.

### DISCUSSION THREAD

**Discussion by F3n67u** [rome/tools#2375] [Suggestions]
Motivation:

my personal preference is single quote instead of double quote, and single quote is very popular in js community so I think it's worth supporting using single quote instead of a double quote.

this can achieved by support an option like prettier's `singleQuote`

https://prettier.io/docs/en/options.html#quotes

**Comment by ematipico** [comment]
> single quote is very popular in js community

Is there some reference about it? 

Do you have some other reasoning why single quotes should be a default? Or, are you suggesting to **introduce** a new option for supporting different type of quotes?

**Reply by F3n67u** [reply]
just support a option let me use single quote insteadof double quote

**Reply by F3n67u** [reply]
For example: 
1. Airbnb style guide prefers “Single Quotes”: https://github.com/airbnb/javascript#strings--quotes
2. react using single quote: https://github.com/facebook/react/blob/1f7a901d7b2fc073e29a1887553b1bd18080da1b/.prettierrc.js#L7
3. jest using single quote: https://github.com/facebook/jest/blob/main/package.json#L125

**Comment by MichaReiser** [comment]
The next release of Rome supports single quotes thanks to @cpojer.

https://github.com/rome/tools/pull/2380
